src/common/models/data-cube/data-cube.ts

```diff
@@ -302,10 +302,6 @@ function readTimeAttribute(config: DataCubeJS, cluster: Cluster | undefined, dim
       throw new Error(`DataCube "${config.name}" must have defined timeAttribute property`);
     }
     const timeAttribute = $(config.timeAttribute);
-    const timeDimension = findDimensionByExpression(dimensions, timeAttribute);
```

Comment:
We should create issue about this change and revisit it later. Maybe sometimes this check makes sens?

Reply:
We added this check previously and didn't test loading json file. This is a fix for old regression really.

Reply:
Added task here: #935